feat/feature.go

```diff
@@ -140,33 +165,42 @@ func PositionWithin(f, ref Feature, position int) (pos int, ok bool) {
 	panic("feat: feature chain too long")
 }
 
-// BaseOrientationOf returns the orientation of the given feature relative to
-// the deepest orientable location and the reference feature, which may be
-// the feature itself if it is not an Orienter or has a nil Location.
-// The returned orientation will always be Forward or Reverse.
+// BaseOrientationOf returns the orientation of f relative to the shallowest
```

Comment:
I'd like to avoid "shallow" and "parent" if possible. "...the first non-nil, non-orientable feature location,..."